dev/src/Command/AddComponentCommand.php

```diff
@@ -201,4 +242,73 @@ private function loadProtoContent(string $proto): string
         $response = $client->get($protoUrl);
         return (string) $response->getBody();
     }
+
+    private function bazelBuildLibrary(string $libraryName, string $googleApisDir): bool
+    {
+        $command = 'which bazel';
+        $output = exec($command, $output, $return);
+        if (strlen($output) == 0) {
+            exit("Error: Bazel is not installed. Please install Bazel before running this script.\n");
+        }
+        chdir($googleApisDir);
+        $command = "bazel query 'filter(\"-(php)$\", kind(\"rule\", //google/cloud/" . $libraryName .
+            "/...:*))' | grep -v -E \":(proto|grpc|gapic)-.*-php$\"";
+        exec($command, $output, $return);
+        if ($return === 0 && !empty($output)) {
+            $command = "bazel build " . $output[0];
+            exec($command, $output, $return);
+            if ($return === 0) {
+                chdir($this->rootPath);
+                return true;
+            }
+        }
+        chdir($this->rootPath);
+        exit("Something went wrong at bazelBuildLibrary, Please use the add-component command instead.\n");
+    }
+
+    private function copyBazelOutToGoogleCloudPhp(string $componentName, string $googleApisDir): bool
+    {
+        $command = 'which docker';
+        $output = exec($command, $output, $return);
+        if (strlen($output) == 0) {
+            exit("Error: Docker is not installed. Please install Docker before running this script.\n");
+        }
+        $googleApisDir = realpath($googleApisDir);
+        $owlbotImage = "gcr.io/cloud-devrel-public-resources/owlbot-cli:latest";
+        $command = "docker run --rm --user $(id -u):$(id -g) -v " . $this->rootPath .
+        ":/repo -v " . $googleApisDir . "/bazel-bin:/bazel-bin " . $owlbotImage .
+        " copy-bazel-bin --config-file=" . $componentName . "/.OwlBot.yaml --source-dir /bazel-bin --dest /repo >&2";
+        exec($command, $output, $return);
+        if ($return === 0) {
+            return true;
+        }
+        exit("Something went wrong at copyBazelOutToGoogleCloudPhp, Please use the add-component command instead.\n");
```

Comment:
Rather than exit, we should throw exceptions. Also, `copyBazelOutToGoogleCloudPhp` is a private function so I'd suggest saying something like "something went wrong running the owlbot container" or something like that

Reply:
I have used `Process::mustRun` method such that it will throw the error if the status code is not `0`.